Example/StandardIntegrationUITests/StandardIntegrationUITests.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  StandardIntegrationUITests.swift
+//  StandardIntegrationUITests
+//
+//  Created by David Estes on 8/21/19.
+//  Copyright © 2019 Stripe. All rights reserved.
+//
+
+import XCTest
+
+class StandardIntegrationUITests: XCTestCase {
+
+    override func setUp() {
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        let app = XCUIApplication()
+        let stripePublishableKey = "pk_test_6Q7qTzl8OkUj5K5ArgayVsFD00Sa5AHMj3"
+        let backendBaseURL = "https://stripe-mobile-test-backend.herokuapp.com/"
+        app.launchArguments.append(contentsOf: ["-StripePublishableKey", stripePublishableKey, "-StripeBackendBaseURL", backendBaseURL])
+        app.launch()
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+    
+    func disableAddressEntry(_ app: XCUIApplication) {
+        app.navigationBars["Emoji Apparel"].buttons["Settings"].tap()
+        app.tables.children(matching: .cell).element(boundBy: 8).staticTexts["None"].tap()
+        app.navigationBars["Settings"].buttons["Done"].tap()
+    }
+    
+    func selectItems(_ app: XCUIApplication) {
+        let cellsQuery = app.collectionViews.cells
+        cellsQuery.otherElements.containing(.staticText, identifier:"👠").element.tap()
+        app.collectionViews/*@START_MENU_TOKEN@*/.staticTexts["👞"]/*[[".cells.staticTexts[\"👞\"]",".staticTexts[\"👞\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
```

Comment:
what's the funky comments here and elsewhere, auto generated artifacts?

Reply:
Xcode adds these during recording, I'll clean them up.

> I have vague fears about these being flakey / taking too long, but I think that's mostly scars from my old gig 😂 .

I do too, but at the moment they're faster than our other tests. We'll monitor and see how it goes.